ansible/README-cloud.md

```diff
@@ -37,6 +37,8 @@
   
 - Optional: Add any other user public keys to allow those users to ssh into the target machine
```

Comment:
Can you add this `... trusted by third party agent as Github to allow those users ...` please ?

Reply:
That would create confusion IMO. 

With that wording it would seem like the playbooks supports various third parties when it in fact does not (since the only significant third party that is support by `ssh-import-it ` is GitHub - the other one is `Launchpad` which is Ubuntu specific)

Reply:
But it should be mentioned that the playbook will then download the public from github using as key the username

Reply:
Doesn't the sentence The usernames supplied need to be valid GitHub usernames that have Public Keys uploaded to their accounts say exactly that?

Reply:
In any case, I updated the docs further, please check them out and let me know if you agree